app/Models/Author.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Author extends Model
+{
+    // use HasFactory;
+    protected $table = 'author';
+    protected $primaryKey = 'author_id';
+    protected $guarded = [];
+
+    public function post(){
+        return $this->hasMany('App\Models\Post','post_author');
+    }
```

Comment:
```suggestion
    public function post()
    {
        return $this->hasMany(Post::class, 'post_author');
    }
```
viết use Post ở trên cùng